docs/api.rst

```diff
@@ -255,7 +253,7 @@ Handling user creation, sign-in, log-out and querying
           "action": "logout"
       }
 
-   :reqjson string action: The action to perform. Can only be one of ``"login"`` or ``"logout"`` and for the logout case has to be ``"logout"``
+   :reqjson string action: The action to perform. Can only be ``"logout"``.
```

Comment:
or empty? So `None` ?

Reply:
To log out, you've to specify `logout` as the action. 

Not passing an action means the user wants to set a premium credential which is stated subsequently in this documentation.